src/app/static/src/app/store/adr/actions.ts

```diff
@@ -146,24 +145,30 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
 
     async uploadFilesToADR(context, uploadFilesPayload) {
         const {state, rootState, commit} = context;
-        const uploadMetadata = rootState.modelCalibrate.uploadMetadata
+        const uploadMetadata = rootState.modelRun.result?.uploadMetadata
         const selectedDatasetId = rootState.baseline.selectedDataset?.id;
         const modelCalibrateId = rootState.modelCalibrate.calibrateId;
         commit({type: ADRMutation.ADRUploadStarted});
 
         for (let i = 0; i < uploadFilesPayload.length; i++) {
-            const { resourceType, resourceFilename, resourceId, displayName } = uploadFilesPayload[i]
+            const {resourceType, resourceFilename, resourceId} = uploadFilesPayload[i]
 
             const requestParams: Dict<string> = {resourceFileName: resourceFilename}
-            if (resourceId){
+            if (resourceId) {
                 requestParams["resourceId"] = resourceId
             }
 
-            if (displayName === "outputSummary") {
+            if (uploadMetadata && resourceType === "outputSummary") {
                 requestParams["description"] = uploadMetadata.outputSummary.description
-            }
-            if (displayName === "outputZip") {
+            } else if (uploadMetadata && resourceType === "outputZip") {
                 requestParams["description"] = uploadMetadata.outputZip.description
+            } else {
+                /**
+                 * Default description will tentatively solve backward
+                 * compatibility issue that may arise with previously calibrated model
+                 * and once re-calibrated, the correct data will be populated
+                 */
+                requestParams["description"] = "Naomi model outputs"
```

Comment:
As @mwoodbri says, a better approach here would probably be to fail and to prompt the user to re-calibrate in error message. 

Also, a heads up that both this and some of the code in ADRController around description is going to conflict with the branch I'm currently working on to add upload of input files, where description is not a required parameter since for input resources description is not changed when new file version is uploaded (though again here I'm going to ask @r-ash if that's expected behaviour!)